README.md

```diff
@@ -60,7 +60,14 @@ class DownloadJob implements Yiisoft\Yii\Queue\Payload\PayloadInterface
 
     public function getData()
     {
-        file_put_contents($this->file, file_get_contents($this->url));
+        return function() {
+            file_put_contents($this->file, file_get_contents($this->url));
+        };
+    }
```

Comment:
This won't work in the most cases, because closures are not serializable.

Reply:
`getData` method should provide serializable data which can be passed through a queue server. And conceptually it should be as simple as a text message, whereby this message can be consumed anywhere else, e.g. in a golang consumer.
Worker handlers are meant to be a place of business-logic implementation.

In the case described above, we should return the given `url` from the `getData` method and create a handler with `file_put_contents` for the `earlyDefinedQueueHandlerName` (see the payload `getName` method) message.